### DISCUSSION TITLE
Wallet connect websocket error 1006 doesnt allow me to build my nextjs app

### DISCUSSION THREAD

**Discussion by arash-hk** [WalletConnect/walletconnect-monorepo#2500] [Developer Support]
hi wallet connect users and  developers, hope you have a greate time .

i have a problem with my appliaction and now our team is going to build the application on the server and we faced to a problem.

when i was developing my app in next js i had this problem but the problem wasn't stop my development journey, till we wanted to deploy our application on server!

you can see my console tab below:

![3](https://github.com/WalletConnect/walletconnect-monorepo/assets/111232556/e2e2e2f7-e772-40d5-b194-bc0a6e2ef8f3)

 
![2](https://github.com/WalletConnect/walletconnect-monorepo/assets/111232556/32b1d033-5ca2-441c-ab54-3f2b748be40b)

and the cool point is when i use rainbow connect button and connect my wallet, and clear the console i dont receive websocket connection error anymore!

can anyone help me about whats going on ?
![4](https://github.com/WalletConnect/walletconnect-monorepo/assets/111232556/41e221c2-79bd-493b-bd48-23019b36df05)

and finally when i run yarn build thats ok in my laptop but when i want to build it out a linux server it rejects the build process and the reason log is : build failed because of webpack errors!
thank you so much guys.

**Comment by finessevanes** [comment]
Can you try setting your relayURL to `wss://relay.walletconnect.com` or `wss://relay.walletconnect.org`?
